Makefile

```diff
@@ -82,7 +82,7 @@ envtest:
 	@$(GO) test ./test/envtest
 
 lint:
-	@golangci-lint run --timeout 15m
+	@golangci-lint run --timeout 15m --max-issues-per-linter 0 --max-same-issues 0
```

Comment:
`.golangci.yaml` has a place for these too. would it make sense to keep all "settings" in one place?

Reply:
@mythi thank you for the review! done